pyomo/contrib/trustregion/TRF.py

```diff
@@ -400,14 +400,14 @@ def solve(self, model, degrees_of_freedom_variables,
             The default is 0 (i.e., no basis function rule.)
 
         """
-        self.config = self._CONFIG(kwds.pop('options', {}))
-        self.config.set_value(kwds)
+        self.local_config = self.config(kwds.pop('options', {}))
+        self.local_config.set_value(kwds)
```

Comment:
Why assign the ephemeral `local_config` to the instance?  It is never referenced outside of this method, and assigning it to the instance makes it less ephemeral...